src/obfl-specification.html

```diff
@@ -1295,7 +1297,7 @@ <h4 id="L836" class="include">page-number</h4>
 <div class="sidebar markup">
 <dl>
   <dt>Usage</dt>
-    <dd>before, after, toc-entry, block, item, style</dd>
+    <dd>before, after, toc-entry, block, toc-block, item, style</dd>
```

Comment:
As ```toc-block``` is used for nesting and text is not allowed then ```page-number``` is not required or needed.